packages/web-sdk/src/instrumentations/webVitals/webVitalsWithAttribution.ts

```diff
@@ -28,114 +31,132 @@ export class WebVitalsWithAttribution {
   }
 
   private measureCLS(): void {
-    onCLS((metric) => {
-      const { loadState, largestShiftValue, largestShiftTime, largestShiftTarget } = metric.attribution;
-
-      const values = this.buildInitialValues(metric);
-      this.addIfPresent(values, 'largest_shift_value', largestShiftValue);
-      this.addIfPresent(values, 'largest_shift_time', largestShiftTime);
-
-      const context = this.buildInitialContext(metric);
-      this.addIfPresent(context, loadStateKey, loadState);
-      this.addIfPresent(context, 'largest_shift_target', largestShiftTarget);
-
-      this.pushMeasurement(values, context);
-    });
+    onCLS(
+      (metric) => {
+        const { loadState, largestShiftValue, largestShiftTime, largestShiftTarget } = metric.attribution;
+
+        const values = this.buildInitialValues(metric);
+        this.addIfPresent(values, 'largest_shift_value', largestShiftValue);
+        this.addIfPresent(values, 'largest_shift_time', largestShiftTime);
+
+        const context = this.buildInitialContext(metric);
+        this.addIfPresent(context, loadStateKey, loadState);
+        this.addIfPresent(context, 'largest_shift_target', largestShiftTarget);
+
+        this.pushMeasurement(values, context);
+      },
+      { reportAllChanges: this.reportAllChanges }
```

Comment:
a lot of changes in these files, but all that I really added was `{ reportAllChanges: this.reportAllChanges }`